tests/unit_tests/hyperion/experiment_plans/test_flyscan_xray_centre_plan.py

```diff
@@ -234,12 +234,14 @@ def test_read_hardware_for_ispyb_updates_from_ophyd_devices(
 
         xgap_test_value = 0.1234
         ygap_test_value = 0.2345
-        ap_sg_test_value = {
-            "name": "Small",
-            "GDA_name": "SMALL_APERTURE",
-            "radius_microns": 20,
-            "location": (10, 11, 2, 13, 14),
-        }
+        ap_sg_test_value = AperturePosition(
+            aperture_x=0,
+            aperture_y=1,
+            aperture_z=2,
+            scatterguard_x=3,
+            scatterguard_y=4,
+            radius=100,
+        )
```

Comment:
```suggestion
        ap_sg_test_value = AperturePosition(
            aperture_x=10,
            aperture_y=11,
            aperture_z=2,
            scatterguard_x=13,
            scatterguard_y=14,
            radius=20,
        )
```

Why is this test no longer setting the AperturePosition to Small?